_data-prepper/pipelines/configuration/sources/opensearch.md

```diff
@@ -106,6 +106,8 @@ Option | Required | Type    | Description
 `indices` | No | Object | The configuration for filtering which indexes are processed. Defaults to all indexes, including system indexes. For more information, see [indexes](#indices).
 `scheduling` | No | Object | The scheduling configuration. For more information, see [Scheduling](#scheduling).
 `search_options` | No | Object | A list of search options performed by the source. For more information, see [Search options](#search_options).
+`serverless` | No | Boolean | Determines whether the OpenSearch Backend is Amazon OpenSearch Serverless. When `true`, ISM policies are not supported in the Amazon OpenSearch Service, therefore any ISM-related configurations set in the sink will have no effect. Default is `false`.
```

Comment:
Same comments as with the sink. Some wording may need to change however since this is the source, not sink.